src/frontend/app/store/selectors/create-service-instance.selectors.ts

```diff
@@ -0,0 +1,25 @@
+import { compose } from '@ngrx/store';
+
+import { AppState } from '../app-state';
+import { CreateServiceInstanceState } from '../types/create-service-instance.types';
+
+export const selectCreateServiceInstance = (state: AppState) => state.createServiceInstance;
+
+export const getOrgGuid = (state: CreateServiceInstanceState) => state.orgGuid;
```

Comment:
`getOrgGuid`, `getSpaceGuid`, `selectOrgGuid` and `selectSpaceGuid` are really generic names that might cause auto-complete confusion.